pkg/bootstrap/testdata/runningsds_golden.json

```diff
@@ -3,9 +3,11 @@
     "id": "sidecar~1.2.3.4~foo~bar",
     "cluster": "istio-proxy",
     "locality": {
-      "region": "regionA",
-      "zone": "zoneB",
-      "sub_zone": "sub_zoneC",
+      "region": "regionA"
+      ,
```

Comment:
is it legal to have comma on new line?

Reply:
This is consistent with the JSON specification，we can use  [json tools](https://jsonlint.com/) to validate it.
![image](https://user-images.githubusercontent.com/8215757/79527411-20b0d480-809a-11ea-8771-1d33eb8a420d.png)